analytics/fresh_refresh.py

```diff
@@ -0,0 +1,126 @@
+# %%
+import pandas as pd
+import os
+from github import Github
+from re import match
+
+pd.set_option('display.max_rows', None)
+### Do not truncate text
+pd.set_option('display.max_colwidth', None)
+
+# %%
+# pygithub object
+g = Github()
+repo = g.get_repo("OpenDataScotland/jkan")
+branch = repo.get_branch("gh-pages")
+last_commit = branch.commit
+#last_commit = repo.get_commit("c7812a7d9135f73256c96ea71863f72dc5a43cc1")
+
+### if running locally, remember to pull latest /jkan/gh-pages also!!!!
+
+# %%
+committed_files = pd.DataFrame()
+for f in last_commit.raw_data['files']:
+    change_type = f['status']
+    changes_headers = None
+    if change_type=='modified':
+        ### breaks the preview into lines
+        extract_lines = f['patch'].split('\n')
+        ### retains only lines with changes (first char is + or -)
+        changes = [line for line in extract_lines if ((line[0]=='+') | (line[0]=='-'))]
+        ### retains only the headers of those lines, drop the first char
+        changes_headers = [line.split(':')[0][1:] for line in changes]
+        ### remove dupe headers
+        changes_headers = list(set(changes_headers))
+
+    temp = pd.DataFrame({'change_type':change_type,
+            'file':[f['filename']],
+            'change':[changes_headers]
+    })
+    committed_files = pd.concat([committed_files,temp], ignore_index= True, axis=0)
+
+committed_files.loc[committed_files['change_type']=='renamed','change_type'] = 'modified'
+print(f'total files: {len(committed_files)}')
+
+# %%
+### Load data from all files in folder
+org_twitter_all = {}
+folder = '../jkan/_organizations'
```

Comment:
You might find it more reliable to get the org details from https://opendata.scot/organizations.json.

If not, I think this path needs to be changed to `../../jkan/_organizations` given that this script is nested within the `analytics` folder